.eslintignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-build
\ No newline at end of file
+build
+testUtils
```

Comment:
I can fix the testUtils. I am not sure about the spec files because there are things it complains about (such as not understanding where `it` and `describe` come from. I am sure there is a way around it but I haven't taken the time to figure it out yet.